rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(data, book, person)
+    @data = data
+    @person = person
+    @book = book
+    person.rental << self
+    book.rental << self
```

Comment:
- [ ] Kindly change from `person.rental` and `book.rental` to `person.rentals` and `book.rentals` respectively. Please remember that we are accessing the `rentals` array attribute in both the `Person` and `Book` classes. The rentals will be an array that consists of records of a particular `Rental` object created by each `Book` or `Person` object. 👍

Reply:
Understood , thanks for this feedback